### PR TITLE
unit test

### PR DESCRIPTION
**Pull request by huuanh99**
## Related Tickets:
- [#45014](https://edu-redmine.sun-asterisk.vn/issues/45014)

## WHAT this PR does?
make unit test for new feature

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by huuanh99** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -93,9 +91,8 @@ public function insertOrder(Request $request)
         $request->session()->forget('cart');
         $request->session()->forget('subtotal');
         $request->session()->forget('count');
-        $request->session()->flash('success', __('ordersuccess'));
         $data = [];
-        $data['content'] = "You just receive an order from " . Auth::user()->name . ", check it out";
+        $data['content'] = "You just receive an order, check it out";
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: localization

File: tests/Unit/Http/controller/OrderControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tests\Unit\Http\controller;
```
Comment (ghost) [inline tests/Unit/Http/controller/OrderControllerTest.php]:
```suggestion
namespace Tests\Unit\Http\Controller;
```

**Comment by huuanh99** [comment]
ready

File: resources/views/admin/layout.blade.php [hunk 3]
```diff
@@ -48,7 +48,11 @@
                         </button>
                         <div class="dropdown-content-notify notifications">
                             @foreach ($notifications as $item)
-                                <a href="{{ route('showOrderPendingView') }}">{{ $item->content }}</a>
+                                @if ($item->type == 1)
```
Comment (ghost) [inline resources/views/admin/layout.blade.php]: bỏ type "1" này vào config nha

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
commented

**Comment by huuanh99** [comment]
ready

**Pull request merged**